lib/rules/no-trailing-spaces.js

```diff
@@ -87,10 +101,14 @@ module.exports = {
                 const re = new RegExp(NONBLANK),
                     skipMatch = new RegExp(SKIP_BLANK),
                     lines = sourceCode.lines,
-                    linebreaks = sourceCode.getText().match(astUtils.createGlobalLinebreakMatcher());
+                    linebreaks = sourceCode.getText().match(/\r\n|\r|\n|\u2028|\u2029/g),
```

Comment:
Any particular reason you decided not to use the regex provided by ast-utils? From a maintenance perspective, it's really nice to have it all in one place.